topi/tests/python/test_topi_math.py

```diff
@@ -18,10 +18,11 @@ def test_ewise():
 
     shape = (20, 3)
 
-    def test_apply(func, name, f_numpy, low, high, check_round=False):
+    def test_apply(func, name, f_numpy, low, high, check_round=False, skip_name_check=False):
         B = func(A)
         assert tuple(B.shape) == tuple(A.shape)
-        assert B.op.body[0].name == name
+        if not skip_name_check:
```

Comment:
@ashutoshparkhi what is the name we are getting here ?

Reply:
sign is implemented with tvm::select inside TOPI. It errors out accessing the name:

AttributeError: '<class 'tvm.expr.Select'>' object has no attribute 'name'

Reply:
ok